io/plugins/eu.esdihumboldt.hale.io.jdbc.spatialite/src/eu/esdihumboldt/hale/io/jdbc/spatialite/reader/internal/SpatiaLiteSchemaReader.java

```diff
@@ -76,4 +80,14 @@ public void setSource(LocatableInputSupplier<? extends InputStream> source) {
 		this.source = source;
 		internalProvider.setSource(new SpatiaLiteJdbcIOSupplier(new File(source.getLocation())));
 	}
+
+	/**
+	 * @see eu.esdihumboldt.hale.common.core.io.IOProvider#execute(eu.esdihumboldt.hale.common.core.io.ProgressIndicator,
+	 *      java.lang.String)
+	 */
+	@Override
+	public IOReport execute(ProgressIndicator progress, String resourceIdentifier)
+			throws IOProviderConfigurationException, IOException {
+		return null;
```

Comment:
Same as above

Reply:
Code modified to throw `UnsupportedOperationException`.